src/target/linux.rs

```diff
@@ -1,63 +1,39 @@
-use nix::{
-    sys::ptrace,
-    sys::wait::waitpid,
-    unistd::{execv, fork, getpid, ForkResult, Pid},
-};
+use nix::unistd::{getpid, Pid};
 use std::{
-    ffi::CString,
     fs::File,
     io::{BufRead, BufReader},
     mem,
 };
 
-/// This structure holds the state of a debuggee.
+use crate::target::unix::UnixTarget;
```

Comment:
I think this can be replaced with

```
use crate::target::unix::{self, UnixTarget};
```

and then on line 25 we can use a more concise module name:

```
let pid = unix::launch(path)?;
```

What do you think?

Reply:
Yeah, that makes it neat.
Is there any configuration for clippy that would capture such lints/best practices?

Reply:
As far as I am aware, clippy doesn't catch that. It's more of a minor code style preference, I guess.